packages/data-store/src/__tests__/contact.store.test.ts

```diff
@@ -1208,15 +1217,29 @@ describe('Contact store tests', (): void => {
             correlationId: 'example_did3',
           },
         },
+        {
+          alias: 'test_alias4',
+          origin: IdentityOrigin.EXTERNAL,
+          roles: [CredentialRole.FEDERATION_OPERATOR],
```

Comment:
Probably we should rename this to FEDERATION_TRUST_ANCHOR to be more inline with the spec , as you can have your roots of trust anywhere in the federation.  Doesn't necessarily have to be at the topmost level with the operator. For now I am okay with calling it operator in the UI, but in our datamodel it should use the correct term

Reply:
changed to FEDERATION_TRUST_ANCHOR